Functions.Templates/Templates/KafkaOutput-JavaScript/function.json

```diff
@@ -16,7 +16,7 @@
             "brokerList" : "BrokerList",
             "topic" : "topic",
             "username" : "$ConnectionString",
-            "password" : "%Password%",
+            "password" : "%EventHubsConnectionString%",
```

Comment:
please check if you can remove username and password and only have connection string

Reply:
Hi @pragnagopa 

We can't remove the username and password, since it is designed for KafkaAPI. 
`ConnectionString` is used for EventHubs only.  Other kafka usage uses user/password (or credentials) 
So that the parameter name should be `username` and `password`  I understand people confuse the naming. 

I just want to avoid `%Password%` since it looks reserved.  How about change it form `%EventHubsConnectionString%` to `%KafkaPassword%`?  Do you think it is less confusing?